BlockServer/epics/archiver_manager.py

```diff
@@ -93,4 +96,13 @@ def _generate_archive_channel(self, group, block_prefix, block):
             else:
                 period.text = str(datetime.timedelta(seconds=1))
                 monitor = eTree.SubElement(channel, 'monitor')
-                monitor.text = str(block.log_deadband)
\ No newline at end of file
+                monitor.text = str(block.log_deadband)
+        else:
+            # Blocks that aren't logged, but are needed for the dataweb view
+            channel = eTree.SubElement(dataweb, 'channel')
+            name = eTree.SubElement(channel, 'name')
+            name.text = block_prefix + block.name
+            period = eTree.SubElement(channel, 'period')
+            period.text = str(datetime.timedelta(seconds=5))
```

Comment:
That suggestion does indeed have the anticipated effect, the current value stays up to date, the last archived value is every 5 minutes or so, I shall change the code in the pull request to reflect this.